core-services/prow/02_config/openshift-assisted/_pluginconfig.yaml

```diff
@@ -61,3 +61,8 @@ plugins:
     - jira
     - size
     - welcome
+triggers:
+- repos:
+  - openshift-assisted
+  trusted_apps:
+  - dependabot
```

Comment:
tip: change IDE/vim settings to add blank lines at end of files
it will save you a lot of time with repos that we usually use as most of them rely on it

Reply:
I'm deleting it manually - usaully linter's complaining about empty line not requiring one 
i'm doing it on auto drive mode :)